app/jobs/export_job.rb

```diff
@@ -42,7 +42,11 @@ def perform(user_id, export_type)
     existing_download = user.downloads.find_by(export_type: export_type)
     existing_download&.destroy!
     download = Download.new(user_id: user_id, contents: data, filename: filename, lookup: SecureRandom.uuid, export_type: export_type)
-    ActiveRecord::Base.logger.silence do
+    if ENV['RAILS_ENABLE_TEST_LOG']
```

Comment:
Can you explain this a bit more? I'm seeing the massive `INSERT INTO downloads` in my logs locally again, which we want to silence.

Reply:
oh whoops I am pretty sure I meant unless